controllers/steps/install/validationStep.go

```diff
@@ -99,6 +101,13 @@ func (vs *ValidationStep) Run(ctx context.Context, doguResource *v2.Dogu) steps.
 	return steps.Continue()
 }
 
+func (vs *ValidationStep) shouldValidateDependencies(doguResource *v2.Dogu) bool {
+	if doguResource != nil && doguResource.Status.Stopped {
```

Comment:
```suggestion
	if doguResource != nil && doguResource.Spec.Stopped {
```

I think this should be the "Spec"